sky/packages/sky/lib/src/material/drawer_item.dart

```diff
@@ -51,8 +49,8 @@ class _DrawerItemState extends State<DrawerItem> {
 
   ui.ColorFilter _getColorFilter(ThemeData themeData) {
     if (config.selected)
-      return new ui.ColorFilter.mode(themeData.primaryColor, ui.TransferMode.srcATop);
-    return new ui.ColorFilter.mode(const Color(0x73000000), ui.TransferMode.dstIn);
+      return new ui.ColorFilter.mode(themeData.primaryColor, TransferMode.srcATop);
```

Comment:
The problem is that we have a widget called ColorFilter.  We could either rename or delete that widget, however.